packages/web-components/fast-foundation/src/button/button.ts

```diff
@@ -187,6 +193,17 @@ export class Button extends FormAssociatedButton {
     }
 
     public control: HTMLButtonElement;
+
+    /**
+     * Manually calls the control's focus when delegatesFocus is unsupported
+     */
+    public handleUnsupportedDelegatesFocus(): boolean {
+        if (this.shadowRoot && this.shadowRoot.delegatesFocus === undefined) {
+            this.control.focus();
+            return false;
+        }
+        return true;
+    }
```

Comment:
I think if we call this focus, then we can probably keep from needing to add the `<template>` tag and alter the DOM structure (which would be breaking). That *should* enable us to just do the following:

```html
<fast-button id="button">Foo</fast-button>
```
```ts
const button = document.getElementById("button");

button.focus();
```

Reply:
See my other reply. I definitely would rather do that to keep it simple, but I don't quite know how to avoid messing with the compatible browsers.

Reply:
I think the problem with the current approach is that it fundamentally breaks the template in a way that is not backwards compatible and is ultimately for (important) but small scenario. I think we need to do this in a way that does not alter the template, which adds complexity but makes it much easier to treat this as a fix and one which can be removed easily over time.

Reply:
@chrisdholt  does this approach that I suggested make sense? 

"Another approach I considered but felt it might be too wasteful was to add an event listener to the element within the button.ts/anchor.ts I can just listen for the focus event there and then use my focus handler. If that's fine too, that's probably the most straight forward solution."

Reply:
I think this is probably reasonable for the time being - less than ideal, but all of this is less-than-ideal, which is why it's a stopgap :)